library/core/src/macros/mod.rs

```diff
@@ -1206,6 +1206,7 @@ pub(crate) mod builtin {
     /// File 'main.rs':
     ///
     /// ```ignore (cannot-doctest-external-file-dependency)
+    /// #![feature(include_macros)]
```

Comment:
Not strictly needed as this is `ignore`d but if this were a stdlib PR I'd have asked for it lol.